### PR TITLE
2.5.0

### PR DESCRIPTION
**Pull request by francescoinfante**

### CODE REVIEW

File: content/user/content/design/styles/icons.md [hunk 1]
```diff
@@ -1,39 +1,32 @@
...
-- The Cities are represented with a Home icon
-- The Companies are represented with a Rocket icon
-- The Investors are represented with a Diamond icon
-- The Markets are represented with a Institution icon
+In the example above, we have chosen 4 different icons to differentiate the
+nodes according to their category:
+- The `Cities` are represented with a *Home* icon
+- The `Companies` are represented with a *Rocket* icon
+- The `Investors` are represented with a *Diamond* icon
+- The `Markets` are represented with a *Institution* icon
```
Comment (davidrapin) [inline content/user/content/design/styles/icons.md]: fix: an *Institution* icon

File: content/user/content/design/styles/node-color.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We can see all the node categories and, clicking on them, we can see their properties and property values.
+
+Clicking on the desire property we can see all the values from the nodes in the visualization; from here, in two steps, we can change the color based on a value:
+
+1. Click on the *color* icon
+
+![](color_node_icon.png)
+
+2. Select one color from the list or insert one other color on the input using the color name (e.g. `blue`), its `hex` (e.g. `#0000FF`)
```
Comment (davidrapin) [inline content/user/content/design/styles/node-color.md]: replace `hex` with: [`hexadecimal color code`](https://www.google.com/search?q=hexadecimal+color+picker)

File: content/user/content/design/styles/node-size.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
 By default all the nodes have the same size.
+
 It is possible to apply different sizes to nodes depeding of their categories and/or property values.
-This way it will be possible to differentiate nodes with a specific category and/or property.
+This way it's possible to differentiate nodes with a specific category and/or property.
```
Comment (davidrapin) [inline content/user/content/design/styles/node-size.md]: it's => it is

File: content/user/content/design/styles/node-size.md [hunk 4]
```diff
@@ -34,9 +34,9 @@ and apply a size on the last one on the same two steps as to apply a color:
 ![](size_node_picker.png)
 
 Now we can see that the majority of nodes continue showing the standard size and the nodes with
-the biggest value with a new size.
+the biggest value has a different size.
 
-The larger node represent Pinterest.
+The larger node represents Pinterest.
```
Comment (davidrapin) [inline content/user/content/design/styles/node-size.md]: fix: This larger node represents the company "Printerest".

File: content/user/content/design/styles/node-size.md [hunk 5]
```diff
@@ -34,9 +34,9 @@ and apply a size on the last one on the same two steps as to apply a color:
...
 We can quickly identify it as the most successful investment of Andreessen Horowitz.
```
Comment (davidrapin) [inline content/user/content/design/styles/node-size.md]: fix: We can quickly identify it as the most successful investment of investor "Andreessen Horowitz".

File: content/user/content/filter/nodes.md [hunk 6]
```diff
@@ -27,58 +25,54 @@ Filtered category:
...
 
-When we have the property type as numerical the values are aggregated on a chart divided always
-in 21 intervals with the size of the bars representing the amount of node with the value inside
-of the interval.
-
-The numerical property section can be divided in three areas:
-* Numerical filtering
-* Numerical values statistics
-* Missing and invalid values statistics
+For numerical property, the values are aggregated on a chart divided always
```
Comment (davidrapin) [inline content/user/content/filter/nodes.md]: remove: "always"

File: content/user/content/filter/nodes.md [hunk 7]
```diff
@@ -27,58 +25,54 @@ Filtered category:
...
+in 21 intervals with the height of the bars representing the amount of nodes with the value inside
+the interval.
 
 ![](numerical_values.png)
 
-#### Numerical filtering:
+#### Numerical filtering
 
-Where we can create a filter to all the values smaller than the lower bound and/or bigger than the upper bound.
+Here we can filter out all the values smaller than the lower bound and/or bigger than the upper bound:
```
Comment (davidrapin) [inline content/user/content/filter/nodes.md]: and/or => and

File: content/user/content/filter/nodes.md [hunk 8]
```diff
@@ -27,58 +25,54 @@ Filtered category:
...
-on the bottom of the chart.
+To activate the interval filtering, we need to activate the lower and/or upper bound by clicking on the `+` button
+of the bound we want to enable.
+ 
+The picker for each bound will then appear on the chart:
 
 ![](numerical_filtering_ative.png)
 
-Dragging the pickers or changing the values on the input is possible to add / update a filter.
+The filter can be updated by dragging the pickers or changing the value directly:
```
Comment (davidrapin) [inline content/user/content/filter/nodes.md]: fix: ... or changing the value directly in the input field:

File: content/user/content/filter/nodes.md [hunk 9]
```diff
@@ -27,58 +25,54 @@ Filtered category:
...
 
 ![](numerical_filtering_filtered.png)
 
-To remove the filter we need to click on the ```trash can``` button of the active bound.
+To remove the filter we need to click on the `trash` button of the active bound.
```
Comment (davidrapin) [inline content/user/content/filter/nodes.md]: fix: To remove a bound, click on the ` trash` icon next to the bound you want to remove.

File: content/user/content/filter/types/types.md [hunk 10]
```diff
@@ -1,30 +1,27 @@
-Before we advance for the filters is essential understand the meaning and
-the importance of the property types.
+Before we move to filters is essential to understand the meaning of property types.
 
-Each property present it's own type on the same line
+Each property shows its own type on the same line:
```
Comment (davidrapin) [inline content/user/content/filter/types/types.md]: fix: For each property, its detected type (textual or numerical) is shown as an icon next to the property name:

File: content/user/content/filter/types/types.md [hunk 11]
```diff
@@ -1,30 +1,27 @@
...
-Numerical type:
+- Numerical type
 
 ![](type_number_auto.png)
 
-The type presented initially is automatically generated and can by used like that
-or set by the user at any time clicking on the type.
-
-Type selection:
+The type is automatically configured and can by used like that
```
Comment (davidrapin) [inline content/user/content/filter/types/types.md]: fix: The type is automatically detected and can by set manually by clicking on the type-icon, this opens the type selector:

File: content/user/content/filter/types/types.md [hunk 12]
```diff
@@ -1,30 +1,27 @@
...
+or reconfigured by the user at any time by clicking on the type:
 
 ![](type_selection.png)
 
-After selection one type the label on the list will change color to show that is set.
-The same thing will happen if one filter is applied on a property value from a type automatically generated.
+After explicitly seleting one type, the label on the list will change color to show that it was manually set.
```
Comment (davidrapin) [inline content/user/content/filter/types/types.md]: fix: After a property-type has been set manually, its icon color will change to red.

File: content/admin/content/viz/styles.md [hunk 13]
```diff
@@ -1,234 +1,214 @@
...
-Visualization styles are defined within the `defaultStyles` and color `palette` key.
-Styles are mapping between visual variables such as colors or size, 
-and data properties on nodes and edges.
+Styles belongs to one particular data-source.
+To set the default styles of a data-source, you have two options:
 
-Visual variables can be used in a single style at a time. 
-For instance we can set `nodes.color` and `nodes.icons`, but we cannot set `nodes.color` twice.
-We can set `nodes.color` and `edges.color`.
+- Configure a style from the workspace and, from the bottom right of the **Design** panel, press **Save Styles ans Captions as Default**
```
[the remark this answers is not in the document]
Comment (francescoinfante) [reply content/admin/content/viz/styles.md]: "and" is not a typo here :p 
`Configure a style from the workspace and, from the bottom right of the **Design** panel, press **Save Styles ans Captions as Default**`looks correct to me.

Show me how you would fix it

File: content/admin/content/viz/styles.md [hunk 14]
```diff
@@ -1,234 +1,214 @@
...
+  "input": ["properties", "name"],
+  "value": "linkurious",
+  "style": {
+    "color": "blue"
+  }
+}
+```
 
-## Color mapping
+The above rule will color all nodes with category `"COMPANY"` and `name="linkurious"` in blue.
```
[the remark this answers is not in the document]
Comment (francescoinfante) [reply content/admin/content/viz/styles.md]: It doesn't make sense to me to talk to the clients about "node.properties.name". 
Ok for the first part but I think it's better to finish with:

... where the `name` is `"linkurious"`.

**Comment by francescoinfante** [comment]
post review done merge to preprod

**Pull request merged**